docs/concepts/variables.md

```diff
@@ -0,0 +1,90 @@
+---
+description: Prefect variables are dynamic-named, mutable string values, much like environment variables.
+tags:
+    - variables
+    - blocks
+---
+
+# Variables
+
+Variables enable you to store and reuse non-sensitive bits of data, often configuration information. Variables are dynamic-named, mutable string values, much like environment variables. Variables are scoped to a Prefect Server instance or a single workspace in Prefect Cloud.
```

Comment:
"dynamic-named" confuses me as a term - do you mean "named, mutable string values much like environment variables"?

Reply:
Yeah thats better